contribs/gnofaucet/.env.example

```diff
@@ -12,4 +12,18 @@ GH_CLIENT_SECRET=
 # For more information on how to set up Redis, see https://redis.io
 REDIS_ADDR=
 REDIS_USER=
-REDIS_PASSWORD=
\ No newline at end of file
+REDIS_PASSWORD=
+
+GH_CLIENT_SECRET=
+GH_APP_ID= 
+GH_CLIENT_ID= 
+GH_APP_PRIVATE_KEY_PATH= 
+
+GH_FETCHER_PR_FACTOR=2
+GH_FETCHER_REVIEW_FACTOR=2
+GH_FETCHER_ISSUE_FACTOR=0.5
+GH_FETCHER_COMMIT_FACTOR=10
```

Comment:
Can you document here in a comment or in a separate README how this is calculated?

If I understand correctly from reading the code, it looks like a merged PR will earn 2 ugnot? That's really low, isn't it?

Also, shouldn't we have different factors for each repo? Because, for example, some are less relevant/important than others. Or if a repo doesn't have ⁠`Squash and merge` enabled on its PRs, and a larger number of commits are merged per PR than on a repo that has it configured?

Reply:
+1

Reply:
Not all repos are created equal 😄

Reply:
I didn't have any guidelines about how to score points, the only reference I had was gnolove, so I implemented the scoring system as it was there. Adding different scoring per repo might need some important refactoring of the business logic, so I would prefer to do it on a different PR, having more context about what we want to have.

> If I understand correctly from reading the code, it looks like a merged PR will earn 2 ugnot? That's really low, isn't it?

Factors are configurable for that reason. I don't know the rewards we want to accomplish for people, or in other words, what is the value in GNOTs of a merged PR, or an opened issue.

Reply:
Regarding small amounts, no problem. Indeed we can change that value. My point was mainly that I wasn’t sure I had calculated the correct result from this variable. As long as a comment or README explains how many GNOTs are obtained from these values, It's fine. 😉 

For handling repo-specific rules in another PR, I have no idea about deadlines or the importance of this feature. Maybe we can merge as it is then wait for your the repo specific rules before enabling the reward system on the public faucet?